hasjob/views/admin_filterset.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+from flask import flash, g, abort
+from coaster.views import route, viewdata, UrlForView, ModelView
+from coaster.auth import current_auth
+from baseframe import __
+from baseframe.forms import render_form, render_redirect
+from .. import app
+from ..models import db, Filterset
+from ..forms import FiltersetForm
+
+
+@route('/f')
+class AdminFiltersetView(UrlForView, ModelView):
+    model = Filterset
+
+    def loader(self, kwargs):
+        if 'name' in kwargs:
+            return Filterset.get(g.board, kwargs.get('name'))
+
+    @route('new', methods=['GET', 'POST'])
+    @viewdata(title=__("New"))
+    def new(self):
+        if 'edit-filterset' not in g.board.permissions(current_auth.user):
```

Comment:
You can do `if g.board.current_permissions.edit_filterset`, but it requires permissions to be valid python symbols. Else, do `if 'edit-filterset' in g.board.current_permissions`.